Model/Grid.cs

```diff
@@ -43,44 +45,76 @@ public void RemoveSquare(int row, int column)
 
         public Sequences GetAvailableSequences(int length)
         {
+            var result = GetAvailableHorizontalSequences(length);
+            if (length == 1)
+            {
+                return result;
+            }
             return GetAvailableHorizontalSequences(length);
```

Comment:
Ovdje treba biti:
`return result.Concat(GetAvailableVertical Sequences(length));`
Želimo rezultate za horizontalne i vertikalne nizove nadovezati. U ovom obliku if grananje nema smisla jer u oba slučaja metoda vraća isti rezultat. If smo stavili samo da se metoda GetAvailableVerticalSequences ne poziva nepotrebno jer bi za duljinu 1 obje metode vraćale potpuno iste nizove. 
Kada to ispravite, svi testovi će Vam prolaziti. To je i ideja testova: da odmah vidite da nešto ne valja i da možete lakše identificirati uzrok.